kernelci/config/__init__.py

```diff
@@ -102,9 +142,27 @@ def load(config_path):
     Load all the YAML files found in the configuration directory then create
     a dictionary containing the configuration objects and return it.
 
-    *config_path* is the path to the YAML config directory
+    *config_path* is the path to the YAML config directory or a
+    unified file
+
     """
     if config_path is None:
```

Comment:
Shouldn't that be true for any empty value like `None` or `''`?

```suggestion
    if not config_path:
```

Reply:
I don't think this is introduced by these changes, but I've added this change a separate commit onto the MR, if this is a change we want to make here.